data/members/franciscoribeiro.json

```diff
@@ -0,0 +1,5 @@
+{
+  "Name": "Francisco Ribeiro",
+  "Job":"Programmer",
+  "Photo": "https://github.com/FranciscoRibeiro.png"
+}
```

Comment:
Those lack of newlines, someone's editor is badly configured 😂

Reply:
Ok I will fix it in the next commit :smile: